src/docs/api.html

```diff
@@ -89,6 +90,13 @@ <h3 class='popover-title'></h3>
           <td><code>'body'</code></td>
         </tr>
         <tr>
+          <td>smartPlacement</td>
+          <td>Boolean</td>
+          <td>Allows to change the step placement if there is not enough place to show it.<br>
```

Comment:
Can you change it to "Dynamically reorients the popover by default" (to borrow the term from Bootstrap docs).
Also the docs for the placement option, update it to the one from Bootstrap.